.github/workflows/continuous-deployment.yml

```diff
@@ -113,45 +103,31 @@ jobs:
     if: github.ref == 'refs/heads/main'
 
     steps:
-      - name: Checkout release branch
-        uses: actions/checkout@v3
+      - name: Download code from GitHub
+        uses: actions/checkout@v4.1.1
         with:
           token: ${{ secrets.GH_TOKEN }}
 
-      - name: Set up Node.js version
-        uses: actions/setup-node@v2
-        with:
-          node-version: "18.12.x"
-
-      - uses: pnpm/action-setup@v2.2.4
-        name: Install pnpm
-        id: pnpm-install
+      - name: Install pnpm package manager
+        uses: pnpm/action-setup@v2.4.0
         with:
-          version: 8.3
-          run_install: false
+          version: 8.14.1
```

Comment:
We hebben geen vulnerability alerts voor versions die hierin genoemd worden, daarom zou ik liever `8.14` gebruiken hier zodat we wel automatisch de laatste patch version gebruiken.

Reply:
Geen idee waarom ik hier patch versie bij had gezet. Maar fixed nu.